### PR TITLE
Use ubuntu & debian instead of unix & freebsd

### PR DESCRIPTION
**Pull request by Anipik**
It seems like unix and freebsd are not known rids
```
NETSDK1056: Project is targeting runtime 'freebsd' but did not resolve any runtime-specific packages
NETSDK1056: Project is targeting runtime 'unix-x64' but did not resolve any runtime-specific packages
```

### CODE REVIEW

**Comment by ericstj** [comment]
```
NETSDK1056: Project is targeting runtime 'freebsd' but did not resolve any runtime-specific packages
NETSDK1056: Project is targeting runtime 'unix-x64' but did not resolve any runtime-specific packages
```

This is because we're still testing RID-specific by having a runtime-specific restore.  The error is there from the SDK to tell the user their self-contained app doesn't have a runtime, so they might have mistyped the RID.  This should go away once we start testing RID-specific assets by examining the runtimeTargets against the reference assemblies (rather than the separate RID pass examining the runtime assemblies).

File: src/Microsoft.DotNet.PackageValidation/GetCompatiblePackageTargetFrameworks.cs [hunk 1]
```diff
@@ -140,8 +140,18 @@ public string GetRidsFromPackage(Package package)
             {
                 if (item.AssetType == AssetType.RuntimeAsset)
                 {
-                    if (!rids.Contains(item.Rid + "-x64"))
-                        rids.Add(item.Rid + "-x64");
+                    string testRid = item.Rid;
+                    if (testRid == "unix")
+                    {
+                        testRid = "ubuntu";
```
Comment (ericstj) [inline src/Microsoft.DotNet.PackageValidation/GetCompatiblePackageTargetFrameworks.cs:146]: Consider adding linux and/or osx instead?

File: src/Microsoft.DotNet.PackageValidation/GetCompatiblePackageTargetFrameworks.cs [hunk 2]
```diff
@@ -140,8 +140,18 @@ public string GetRidsFromPackage(Package package)
...
+                    }
+                    else if (testRid == "freebsd")
+                    {
+                        testRid = "debian";
```
Comment (ericstj) [inline src/Microsoft.DotNet.PackageValidation/GetCompatiblePackageTargetFrameworks.cs:150]: What made you choose `debian` here? https://github.com/dotnet/runtime/blob/3eddf4a06b0755747f8e8d659264694eef1f31ee/src/libraries/Microsoft.NETCore.Platforms/pkg/runtimeGroups.props#L59-L60

I'm a bit surprised we have any packages shipping these assets.

Comment (ericstj) [reply src/Microsoft.DotNet.PackageValidation/GetCompatiblePackageTargetFrameworks.cs:150]: Consider handling through https://github.com/dotnet/runtime/blob/89fbdcafcc09af0e1c2d86783eb8e8b93a743968/src/libraries/pkg/test/testPackages.proj#L18

Comment (safern) [reply src/Microsoft.DotNet.PackageValidation/GetCompatiblePackageTargetFrameworks.cs:150]: cc: @wfurt regarding packages shipping freebsd assets.

**Review by ericstj — APPROVED** [review]

File: src/Microsoft.DotNet.PackageValidation/GetCompatiblePackageTargetFrameworks.cs [hunk 3]
```diff
@@ -140,8 +140,21 @@ public string GetRidsFromPackage(Package package)
             {
                 if (item.AssetType == AssetType.RuntimeAsset)
                 {
-                    if (!rids.Contains(item.Rid + "-x64"))
-                        rids.Add(item.Rid + "-x64");
+                    string testRid = item.Rid;
+                    string testArch = "-x64";
+                    if (testRid == "unix")
+                    {
+                        if (!rids.Contains("linux" + testArch))
```
[the remark this answers is not in the document]
Comment (Anipik) [reply src/Microsoft.DotNet.PackageValidation/GetCompatiblePackageTargetFrameworks.cs:147]: I can address this change with all the other feedback in the previous pr.

**Pull request merged**